modules/odc-accessing-perspectives.adoc

```diff
@@ -13,7 +13,7 @@ You can access the *Administrator* and *Developer* perspective from the web cons
 To access a perspective, ensure that you have logged in to the web console. Your default perspective is automatically determined by the permission of the users. The *Administrator* perspective is selected for users with access to all projects, while the *Developer* perspective is selected for users with limited access to their own projects
 
 .Additional Resources
-See link:https://docs.openshift.com/container-platform/4.12/web_console/adding-user-preferences.html[Adding User Preferences] for more information on changing perspectives.
+See link:https://docs.openshift.com/container-platform/4.13/web_console/adding-user-preferences.html[Adding User Preferences] for more information on changing perspectives.
```

Comment:
:( Not related to your PR, but there are so many hard-coded links in here that we should probably fix. CC @kalexand-rh

Reply:
Agreed, this has been an eye-opening experience!